### ISSUE TITLE
Effective 2.4.1-1.21.1 not compatible with Sodium 1.21.1-0.6.5 Fabric

### ISSUE THREAD

**Issue by AnitoWasTaken** [Ladysnake/Effective#287]
When both Effective 2.4.1-1.21.1 and Sodium 1.21.1-0.6.5 fabric are installed it results in a crash upon joining a world. Minecraft crashes on both singleplayer and multiplayer worlds. I have not tried this with different types of mod versions.

Mods:
Effective 2.4.1-1.21.1
Fabric API 0.114.0+1.21.1
Sodium mc1.21.1-0.6.5-fabric

Fabric Version:
0.16.9

I attached the crash report below.
[crash-2024-12-30_21.26.12-client.txt](https://github.com/user-attachments/files/18277626/crash-2024-12-30_21.26.12-client.txt)

**Comment by RoootTheFox** [comment]
i believe the full log is more useful: https://mclo.gs/1cJ0cQw
seems to be a Veil issue from what i can tell? i assume Veil isn't updated to support Sodium 0.6.5, making the compat mixin fail

**Comment by Mitrays** [comment]
Yes, really can't play without sodium. hope it's fixable even when it's Veil issue.

**Comment by doctor4t** [comment]
Fixed in 2.4.4
